webpack.config.js

```diff
@@ -12,15 +12,12 @@ module.exports = {
   devtool: '#cheap-module-eval-source-map',
 
   module: {
-    loaders: [
+    rules: [
       {
-        test: /\.js$/,
+        test: /\.(js)$/,
```

Comment:
Not sure if this was copied from somewhere, but I can't see what difference this change would make?